docs/iperf2.md

```diff
@@ -31,5 +31,7 @@ rm -rf shadow.data; shadow shadow.yaml > shadow.log
 
 ## Notes
 
-1. Shadow doesn't support `setitimer()` so you must build iPerf 2 with `#define
-HAVE_SETITIMER 0` set.
+1. You must use an iPerf 2 version >= `2.1.1`. Older versions of iPerf have a
+[busy loop][busy-loop] that is incompatible with Shadow.
```

Comment:
Maybe be a little more specific here so that it doesn't sound like all busy loops are incompatible with shadow? e.g. "no-syscall busy-loop", or "cpu-only busy-loop"?

Reply:
Added the "no-syscall".